### DISCUSSION TITLE
macOS runners

### DISCUSSION THREAD

**Discussion by vishalmummigatti** [community/community#108363] [Actions]
### Select Topic Area

General

### Body

Are github actions managed macOS runners compliant with Apple's 24 hr minimum allocation period? If so, am I billed for a minimum of 24 hrs? If not compliant, can you explain how github is able to bypass this requirement?https://www.apple.com/legal/sla/docs/macOSSonoma.pdf 

Section 3: Leasing for Permitted Developer Services.

**Comment by DJLink** [comment]
As stated in Github Actions help page, for MacOS consume 10 times the rate that jobs on Linux runners consume. So you are charged per minute but there are different rates for the different machines.


> Jobs that run on Windows and macOS runners that GitHub hosts consume minutes at 2 and 10 times the rate that jobs on Linux runners consume. For example, using 1,000 Windows minutes would consume 2,000 of the minutes included in your account. Using 1,000 macOS minutes, would consume 10,000 minutes included in your account.


https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#minute-multipliers

And the prices per minute are listed here:

https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#per-minute-rates

**Comment by vishalmummigatti** [comment]
@DJLink understood but it's still not meeting Apple's minimum requirement for 24 hours. It seems like the minimum time you will be charged for is still 1 minute (even though the 1 minute macOS price = 10 times the per minute price for a linux runner). I would have thought that you would need to, at a minimum, be charged for 24hrs/day * 60 min/hour = 1,440 minutes for macOS runners to be compliant with macOS EULA.
